lib/OnyxUtils.ts

```diff
@@ -427,7 +427,7 @@ function getPureKey(key: OnyxKey): string {
     if (!key) {
         return '';
     }
-    return key.replace(/\d+/g, '');
+    return key.replace(/_\w+/g, '_');
 }
```

Comment:
I think this is definitely better and should cover most use-cases. But just hypothetically, isn't it possible to have more than one `_` in the collection key or have no `_` at all? Sth. like `reports.` or `reports$`?

Reply:
I think if we would do this, it would be larger change